scripts/close_inactive.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+REPOS = ["theforeman/foreman"]
+KEEP = 6 # months
+THRESHOLD = DateTime.now << KEEP
+COMMENT = <<EOC
+This PR has been inactive for #{KEEP} months, closing for now.
+Feel free to reopen when you return to it.
+Thank you for your contribution, @%s!
+EOC
+LABELS = ["Inactive"]
+
+c = Octokit::Client.new(:access_token => ENV['GITHUB_OAUTH_TOKEN'])
+REPOS.each do |repo|
+  c.pull_requests(repo).each do |pr|
```

Comment:
> > +  c.pull_requests(repo).each do |pr|
> 
> This should probably filter out PRs that are waiting for review, only WoC.

Hmmm looks like it is not possible.

https://developer.github.com/v3/pulls/#list-pull-requests

Need to test search capabilities, its for issues only...

https://developer.github.com/v3/search/#search-issues

-- 
Later,
 Lukas #lzap Zapletal